examples/fomac/example_fomac.cpp

```diff
@@ -224,3 +224,64 @@ auto FoMaC::get_parameters_num(const QDMI_Operation &op) const -> size_t {
   throw_if_error(ret, "Failed to query the parameter number");
   return parameters_num;
 }
+
+auto FoMaC::get_telemetry_sensors() const -> std::vector<QDMI_TelemetrySensor> {
+  size_t telemetry_sensor_size = 0;
+  int ret = QDMI_device_query_device_property(
+      device, QDMI_DEVICE_PROPERTY_TELEMETRYSENSORS, 0, nullptr,
+      &telemetry_sensor_size);
+  throw_if_error(ret, "Failed to get the telemetry variable list size.");
```

Comment:
```suggestion
  throw_if_error(ret, "Failed to get the telemetry sensor list size.");
```
That should not be variable anymore, correct?

Reply:
Corrected